modules/camera/backend/modules/camera/urls.py

```diff
@@ -0,0 +1,13 @@
+
+from django.urls import path, include
+from rest_framework import routers
+
+from .viewsets import ImageViewSet, ImageUploadView
+
+
+router = routers.DefaultRouter()
+router.register(r'photos/user', ImageViewSet)
```

Comment:
This endpoint isn't being used in the RN code. Let's please not include it in the first release if it's not being used.

Reply:
it was part of the documentation I was given.